<?xml version="1.0"?>
<launch>
    <!-- ============================ SETTINGS ============================= -->
    <!--  -->
    <arg
    name="camera_name" 
    default="camera"/>

    <!--  -->
    <arg 
    name="image_rotation" 
    default="-90"/>

    <!--  -->
    <arg 
    name="depth_averaging_enable" 
    default="true"/>

    <!--  -->
    <arg 
    name="depth_averaging_count" 
    default="10"/>

    <!--  -->
    <arg 
    name="coordinates_averaging_enable" 
    default="true"/>

    <!--  -->
    <arg 
    name="coordinates_averaging_count" 
    default="5"/>

    <!-- ======================== END OF SETTINGS ========================== -->
    
    <!-- Pose landmarks estimator: -->     
    <node 
    pkg="data_collector" 
    type="pose_landmarks.py"
    name="pose_landmarks" 
    output="screen">
        <param 
        name="camera_name" 
        value="$(arg camera_name)"/>
        <param 
        name="image_rotation" 
        value="$(arg image_rotation)"/>
        <param 
        name="depth_averaging_enable" 
        value="$(arg depth_averaging_enable)"/>
        <param 
        name="depth_averaging_count" 
        value="$(arg depth_averaging_count)"/>
        <param 
        name="coordinates_averaging_enable" 
        value="$(arg coordinates_averaging_enable)"/>
        <param 
        name="coordinates_averaging_count" 
        value="$(arg coordinates_averaging_count)"/>
    </node>     
</launch>